cmd/config-example.env

```diff
@@ -0,0 +1,8 @@
+ALLOWED_ORIGINS=* # or your frontend url
+ALLOW_CREDENTIALS=true
+ADDRESS_ECHO=:5477 #address echo to start
+
+#Logger
+#if you want to log to file, create a folder called logs in the root of the project.
+LOG_LEVEL=debug #from this level and above will be logged (trace, debug, info, warning, error, fatal, panic)
+LOG_SAVE_FILE=false # if false logs are written in the console
```

Comment:
how this file is going to be used?

Reply:
Typically, in my projects, I avoid pushing the 'config.env' file to the online repository to prevent sensitive information, like database connection passwords, from being exposed. Instead, I create a 'config-example.env' file, containing all the necessary variables for the project along with their explanations on how the project should be configured and run.